docs/HACKING.md

```diff
@@ -1,6 +1,29 @@
 ⚠⚠⚠ THIS IS A LIVING DOCUMENT AND LIKELY TO CHANGE QUICKLY ⚠⚠⚠
 
 # Hacking on the WMCO
+Requires all pre-requisites in the [README](/README.md) to be fufilled.
```

Comment:
We might want to wait until #476 merges then link to the pre-reqs doc

Reply:
Linking to the pre-req doc should be done in #476 wherever needed. IOW, this change can be removed from here but included in #476.